examples/api/lib/material/floating_action_button/floating_action_button.1.dart

```diff
@@ -11,25 +11,27 @@ void main() => runApp(const MyApp());
 class MyApp extends StatelessWidget {
   const MyApp({Key? key}) : super(key: key);
 
-  static const String _title = 'Flutter Code Sample';
+  static const String _title = 'FloatingActionButton Sample';
 
   @override
   Widget build(BuildContext context) {
     return const MaterialApp(
       title: _title,
```

Comment:
We don't actually need an app title (it only sets the app title for the task on Android, I think), so you could eliminate the `_title` variable, and the argument to `FabExample`, and just hard code the string in the AppBar.

(and in the other examples too).